tests/cases/ustruct/LV_NeoHookean_passive_genBC/post_processing_functions.py

```diff
@@ -0,0 +1,700 @@
+# Before running on Sherlock, need to have installed: 
+#
+# pyvista: pip3 install pyvista
+# ffmpy: pip3 install ffmpy
+#
+# Also, need to load the following required modules on Sherlock: 
+#
+# ml system mesa ffmpeg
+import glob
+import os
+import pyvista as pv # for VTK mesh manipulations
+import numpy as np 
+#import ffmpy # to convert .avi movie to .mp4 movie
+
+def get_timestep(result_file):
```

Comment:
Do we need to repeat these python pre/post-processing every single time? I would store them somewhere just once.

Reply:
Following in person discussion with Matteo and Martin, keeping these Python files.